vip-mail.php

```diff
@@ -32,9 +32,11 @@ protected static function isPermittedPath( $path ) {
 		}
 	}
 }
-global $phpmailer;
-$phpmailer = new VIP_PHPMailer( true ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
 
+if ( defined( 'USE_VIP_PHPMAILER' ) && USE_VIP_PHPMAILER ) {
```

Comment:
I always it's better to be explicit with boolean constants.

```suggestion
if ( defined( 'USE_VIP_PHPMAILER' ) && true === USE_VIP_PHPMAILER ) {
```